### ISSUE TITLE
Hope this project could have some update

### ISSUE THREAD

**Issue by stevejobschen** [mackboudreau/DalhousieScheduleBuilder#3]
I'm looking forward to use this website.

**Comment by mack** [comment]
You're absolutely right @stevejobschen! This repo could use some love. I think I'm going to reboot it up this year and do it right this time. I sort of cringe looking back at how poorly I made this. Give me a couple months and I'll have a much better revision.
